src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +90,23 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('ui_test', False):
+            # json object for variables file
+            variables = {}
+            variables['api'] = {}
+
+            # export to variables.json object
+            variables['api'].update({
+                'olympia.dev': dict([
+                    ('username', user.username,),
+                    ('jwt_issuer', apikey.key),
+                    ('jwt_secret', apikey.secret)
+                ])
+            })
+
+            with open('tests/ui/variables.json', 'w') as outfile:
+                json.dump(variables, outfile, indent=2)
+
```

Comment:
Good point, I thought that it'd be simple to parse the stdout the `createsuperuser` script which gives you JSON but this probably is more crazy than just doing it in the script.

I'd change the behaviour here though, let's not print the JSON in https://github.com/mozilla/addons-server/pull/4624/files#diff-b4c3c77fd0d5100ac20e3b52f20e0bb9R86 if `ui-test` option is given and rename `ui-test` to maybe something like `safe-api-credentials` with an argument that actually points to a target (here `tests/ui/variables.json`) so you can control this from external. You should be able to do that with something like (untested)

```
        parser.add_argument(
            '--save-api-credentials',
            dest='save_api_credentials',
            type=str,
            help='Saves the generated API credentials into a JSON file'
        )
```

and then you can use the value of `save_api_credentials` from `options` as an target for `open`